.github/workflows/wheels.yml

```diff
@@ -108,6 +115,4 @@ jobs:
 
       - uses: pypa/gh-action-pypi-publish@release/v1
         with:
-          user: __token__
-          # Remember to generate this and set it in "GitHub Secrets"
-          password: ${{ secrets.pypi_password }}
+          print-hash: true
```

Comment:
I still need to setup the permissions on PyPI (https://docs.pypi.org/trusted-publishers/adding-a-publisher/) but I don't have owner level permissions currently.

Reply:
This is done now.